test-integration/operator-tests/src/main/java/org/infinispan/Infinispan.java

```diff
@@ -102,59 +97,4 @@ private Credentials getCredentials(String secretName, String username) throws IO
       Identities identities = new ObjectMapper(new YAMLFactory()).readValue(identitiesYaml, Identities.class);
       return identities.getCredentials(username);
    }
-
-   public void createServiceMonitor() throws IOException {
```

Comment:
@Crumby can we add `ServiceMonitor` validation to this code?

Reply:
@dmvolod Can you elaborate more on `ServiceMonitor` validation? `MonitoringStackIT` already validates that endpoints are scraped by the Prometheus so there is that validation. I should probably add `ServiceMonitor` to cleanup validator though.

On the identiteis. If you mean the code above, then that code is related user secrets

Reply:
> Can you elaborate more on `ServiceMonitor` validation? `MonitoringStackIT` already validates that endpoints are scraped by the Prometheus so there is that validation. I should probably add `ServiceMonitor` to cleanup validator though.
> 
Oh, I missed that `MonitoringStackIT` validates `ServiceMonitor`.

> On the identities. If you mean the code above, then that code is related user secrets

Yes, I see that you got username and password from `identities.yaml` but now Secret has `username` and `password` separate fields and not need to parse yaml file anymore

Reply:
@dmvolod But that applies only for the Operator credentials, right? The tests themselves are working with user credentials which are still only provided via `identities.yaml`

Reply:
Yeah, @Crumby right, you are testing from the customer point of view.